android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/home/HomeScreen.kt

```diff
@@ -30,27 +36,48 @@ fun HomeScreen(viewModel: HomeViewModel = getViewModel()) {
         )
     }
 
-    HomeScreenContent(currentDate, avatarUrlString, isLoading)
+    // TODO: Update survey data later
+    val uiModel = HomeUiModel(
+        currentDate,
+        avatarUrlString,
+        HomeSurveysUiModel(
+            "Working from home Check-In Check-In",
+            "We would like to know what are your goals and skills you wanted!",
+            "https://dhdbhh0jsld0o.cloudfront.net/m/1ea51560991bcb7d00d0_l",
+            3,
+            1,
+            isLoading
+        ),
+        isLoading
```

Comment:
So that `HomeUiModel.isLoading` is considered header loading? I think having a new `HomeHeaderUiModel` or renaming it to `isHeaderLoading` makes more sense.

```suggestion
        HomeHeaderUiModel(
            currentDate,
            avatarUrlString,
            isLoading
        ),
        HomeSurveysUiModel(
            "Working from home Check-In Check-In",
            "We would like to know what are your goals and skills you wanted!",
            "https://dhdbhh0jsld0o.cloudfront.net/m/1ea51560991bcb7d00d0_l",
            3,
            1,
            isLoading
        )
```